### DISCUSSION TITLE
Separate font size.

### DISCUSSION THREAD

**Discussion by dmitriy71n** [phcode-dev/phoenix#1798] [Ideas]
![2024-08-09 172156](https://github.com/user-attachments/assets/18a45e2e-326d-4ea8-bde4-0676894cdbad)
1. Separate font size for the vertical Phoenix Code menu or the standard system one.
2. Separate, changeable font size for the editor.

This is very inconvenient when you set the editor font size larger than the default of 16px - 18px and the top Phoenix Code menu becomes huge on the entire screen.

**Comment by abose** [comment]
@dmitriy71n Thanks for reporting the issue. 

Fixed via: https://github.com/phcode-dev/phoenix/pull/1799 . Please see the linked issue for more details on the bug.

Fix will be avilable in the next release.
